ui/js/directives/treeherder/resultsets.js

```diff
@@ -1,48 +0,0 @@
-treeherder.directive('thActionButton', function () {
```

Comment:
This file is in the repo still, but as 0 bytes. Could you remove it plus the others that are also empty? :-)

```
treeherder $ find ui/ -empty
ui/js/directives/treeherder/clonejobs.js
ui/js/directives/treeherder/resultsets.js
ui/js/reactrevisions.jsx
ui/partials/main/thFilterCheckbox.html
```

Reply:
Sheesh, thanks for catching this!  I can't believe I missed them...

Reply:
They display virtually identically in GitHub so hard to spot - think I've ended up with similar 0 bytes files when selectively staging lines in sourcetree sometimes - guess the "stage all lines" isn't the same as "stage file" when removing a file?